### ISSUE TITLE
Error when getting user

### ISSUE THREAD

**Issue by RealMoondancer** [Renondedju/Osu.py#8]
**Describe the bug**
An error is raised when trying to get the user using Python 3.10

**Error report**

```
Traceback (most recent call last):
  File "C:\Users\Admin\.virtualenvs\twitchbot-ckDLcYWF\lib\site-packages\twitchio\client.py", line 193, in wrapped
    await func(*args)
  File "C:\Users\Admin\Documents\twitchbot\bot.py", line 19, in event_ready
    self.too_md = await self.api.get_user("too_md")
  File "C:\Users\Admin\.virtualenvs\twitchbot-ckDLcYWF\lib\site-packages\pyosu\api.py", line 168, in get_user
    return User(user_events=[UserEvent(**event) for event in data.get('events', [])], **data, api=self)
  File "C:\Users\Admin\.virtualenvs\twitchbot-ckDLcYWF\lib\site-packages\pyosu\api.py", line 168, in <listcomp>
    return User(user_events=[UserEvent(**event) for event in data.get('events', [])], **data, api=self)
  File "C:\Users\Admin\.virtualenvs\twitchbot-ckDLcYWF\lib\site-packages\pyosu\models\user_event.py", line 35, in __init__
    self.beatmap_id	    = int(data.get('beatmap_id'	  , 0 ))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

**To Reproduce**

1. Create OsuApi in Python 3.10
2. Await api.get_user("some user")

**Expected behavior**
The Api would return the User Struct

**Python version**
Python 3.10.1

**Comment by Renondedju** [comment]
I cannot reproduce the bug and need further information.  
  
- Does this happen every time?  
- What are you trying to achieve?  
- Does this happen for every user you tried?
- I see you are using some kind of virtual environment, have you tried it on your system instead?

**Comment by RealMoondancer** [comment]
- Yes, this happens every time
- I am trying to make a bot which automatically tracks the user stats (and yes I know there are already some existing)
- Yes, this happens for every user I tried
- Yes, I tried this on my main system and it seems to happen there too

**Comment by Renondedju** [comment]
Just released [v0.5.3](https://github.com/Renondedju/Osu.py/releases/tag/v0.5.3), this should fix the issue.  
The Osu API seems to be sending null values instead of omitting them all together (as it previously did?)  
  
Anyway, make sure to update the lib by running `pip install -U git+https://github.com/Renondedju/Osu.py` on the right python installation. I'm waiting for your validation to close the issue.

**Comment by RealMoondancer** [comment]
Works now, thanks for the fix
